opta/commands/config.py

```diff
@@ -0,0 +1,54 @@
+import click
+from click_didyoumean import DYMGroup
+
+from opta.core.aws import AWS
+from opta.core.cloud_client import CloudClient
+from opta.core.gcp import GCP
+from opta.exceptions import UserErrors
+from opta.utils import logger
+
+
+@click.group(cls=DYMGroup)
+def config() -> None:
+    """
+    Manage opta configurations
+    """
+    pass
+
+
+cloud_option = click.option(
+    "--cloud",
+    help="The cloud provider to use",
+    required=True,
+    type=click.Choice(["aws", "google", "azurerm"]),
+)
+
+
+@config.command()
+@cloud_option
+def view(cloud: str) -> None:
+    """
+    View the opta configuration file for a given cloud provider
+
+    Use the current cloud credentials to fetch the remote opta config file
+    """
+    if cloud.lower() == "azurerm":
+        raise UserErrors("Currently AzureRM isn't supported.")
```

Comment:
nit: Currently Azure is not supported for this comment. As Opta's Azure support is still in Beta